release-notes.md

```diff
@@ -167,6 +167,14 @@ The following updates were made in this release:
 
 These changes were released as part of ACPCore CocoaPod v2.3.6.
 
+## November 8, 2019
+
+The following updates were made in this release:
```

Comment:
since it's only 1 item, please reword to:
"The following update was made in this release:"